docs/installation.md

```diff
@@ -10,74 +10,53 @@ These steps are required once per machine.
 ## Step 1 - sbt plugin
 
 The server needs to access some metadata about the build configuration. This
-data are produced by an sbt plugin. This plugin is currently not published, so
-you will need to copy paste it on your machine.
+data are produced by an sbt plugin.
 
-Here's the source of the plugin:
-https://github.com/scalameta/metals/blob/master/project/MetalsPlugin.scala
+You can install the plugin with
 
-Copy the source to either (depending on your sbt version):
-
-* (sbt 0.13) `~/.sbt/0.13/plugins/MetalsPlugin.scala`
-* (sbt 1.+) `~/.sbt/1.0/plugins/MetalsPlugin.scala`
-
-## Step 2 - build the VSCode extension
-
-The VSCode extension is not yet published on the Marketplace, so you'll need to
-build it locally.
-
-* Make sure you have installed `node`, `npm` and VS Code.
-* `cd vscode-extension`
-* `npm install`
-* `npm run build`
-* `code --install-extension metals-0.1.0.vsix`
-
-## Step 3 - publish the server locally
+```
+addSbtPlugin("org.scalameta" % "sbt-metals" % "<version>")
+```
```

Comment:
Link to the Bintray repo for the available versions?

Reply:
would it be worth to mention that the plugin should best be added into the global `~/.sbt/1.0/plugins/` path?